README.md

```diff
@@ -63,7 +63,8 @@ Read the complete in detail documentation in the [Usage section](https://github.
 
 ## Usage
 
-Currently, the package contains shapes: [Circle](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/circle.tsx), [Donut](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/donut.tsx), [CircleGrid](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/circlegrid.tsx), [Diamond](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/diamond.tsx), [PolygonCard](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/polygonCard.tsx), [Polygon](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/polygon.tsx), [Star](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/star.tsx), [Cross](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/cross.tsx), [SquareDonut](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/squareDonut.tsx), [Arrow](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/arrow.tsx), [Message](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/message.tsx).
+Currently, the package contains shapes: [Circle](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/circle.tsx), [Donut](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/donut.tsx), [CircleGrid](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/circlegrid.tsx), [Diamond](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/diamond.tsx), [PolygonCard](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/polygonCard.tsx), [Polygon](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/polygon.tsx), [Star](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/star.tsx), [Cross](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/cross.tsx),
+[Heart](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/heart.tsx), [SquareDonut](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/squareDonut.tsx), [Arrow](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/arrow.tsx), [Message](https://github.com/ashutosh1919/react-awesome-shapes/blob/main/src/lib/shapes/message.tsx).
```

Comment:
Should be on the same line.

Reply:
sorry I don't get it! would you give more hints